package.json

```diff
@@ -1737,6 +1738,31 @@
                 "mimeTypes": [
                     "application/vnd.jupyter.widget-view+json"
                 ]
+            },
+            {
+                "id": "jupyter-notebook-renderer",
+                "entrypoint": "./out/client_renderer/renderers.js",
```

Comment:
Copy output from renderer extension into jupyter extension & things just work.
During `npm install` & builds on CI we will download the renderer extension output & use the latest. 
This way it works for dev & in builds.